example/hr-data-generator/src/test/java/uk/gov/gchq/palisade/example/hrdatagenerator/EmployeeTest.java

```diff
@@ -30,8 +30,8 @@
     public void generateEmployee() {
         long startTime = System.currentTimeMillis();
         Random random = new Random(0);
-        for (int i = 0; i < 100; i++) {
-            Employee.generate(random);
+        for (int i = 0; i < 1; i++) {
```

Comment:
change this back to:
for (int i = 0; i < 100; i++) {
            Employee.generate(random);